Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -8,12 +8,31 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
+#include <TGeoManager.h>
+#include <TRandom.h>
+
 #include "TRDSimulation/Digitizer.h"
+#include "TRDBase/TRDGeometry.h"
+#include "TRDBase/TRDPadPlane.h"
+
+#include "FairLogger.h"
 
 using namespace o2::trd;
 
 Digitizer::Digitizer()
 {
+  // Check if you need more initialization
+  mGeom = new TRDGeometry();
```

Comment:
Hello @sawenzel, This is one of my concerns, it this all what is required to initialise the TRD Geometry?. I am worried about if this is enough since in the Detector class it is initialized differently, see for example:

```
Detector::Detector(const Detector& rhs)
  : o2::Base::DetImpl<Detector>(rhs),
    mHits(o2::utils::createSimVector<HitType>()),
    mFoilDensity(rhs.mFoilDensity),
    mGasNobleFraction(rhs.mGasNobleFraction),
    mGasDensity(rhs.mGasDensity),
    mGeom(rhs.mGeom),
    mTRon(false)
{
  if (TRDCommonParam::Instance()->IsXenon()) {
    mWion = 23.53; // Ionization energy XeCO2 (85/15)
  } else if (TRDCommonParam::Instance()->IsArgon()) {
    mWion = 27.21; // Ionization energy ArCO2 (82/18)
  } else {
    LOG(FATAL) << "Wrong gas mixture";
    // add hard exit here!
  }
}
```

Reply:
Then, I don't understand how `mGeom` is initialized in Detector.cxx and that would also reveals a bug in the code, since de value assigned to `mWion` should appear in the default constructor too.

Reply:
@sawenzel, In summary, is it enough to initialize `mGeom` in Digitizer.cxx as `mGeom = new TRDGeomertry()`?, or it requieres more steps?

Reply:
What is the functionality that you require from TRDGeometry in the digitizer? Just
some static geometry services? I suspect that just instantiating such an object could be enough ... but that's to be seen.

Reply:
Ok, I think it actually works until now, but I will check. The only non-static method required is 

```
GPUd() const TRDPadPlane* getPadPlane(int layer, int stack) const { return &mPadPlanes[getDetectorSec(layer, stack)]; }
```

It is currently working (compiling and running), but I will put a reminder of testing that this is working ok and returning reasonable values.